CMakeLists.txt

```diff
@@ -44,3 +44,14 @@ if(GREENTEA_CLIENT_DUMMY_METRICS)
             source/greentea_metrics.cpp
     )
 endif()
+
+# Example application
+
+SET(GREENTEA_CLIENT_EXAMPLE OFF CACHE BOOL "Build the example application")
+
+if(GREENTEA_CLIENT_EXAMPLE)
+    add_executable(greentea-client-example example/main.cpp)
+    target_link_libraries(greentea-client-example
+            greentea-client
+    )
+endif()
```

Comment:
Should we move this down into a CMakeLists.txt in the example folder and just add the subdirectory here?

Reply:
Good idea